File: frontend-mealsonwheels/src/types/reassessment/ReassessmentEvaluationDTO.ts
// src/types/management/ReassessmentEvaluationDTO.ts

export interface ReassessmentEvaluationDTO {
  id: number;
  evaluationDate: string; // ISO date-time string
  evaluationNotes?: string;
  serviceAdjustments?: string;
  memberId: number;
  orderId?: number;
  rating?: number;
  comment?: string;
}

File: backend-mealsonwheels/src/main/java/com/merrymeal/mealsonwheels/dto copy/ReassessmentEvaluationDTO.java
package com.merrymeal.mealsonwheels_backend.dto;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.*;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ReassessmentEvaluationDTO {

    private Long id;

    @NotNull(message = "Evaluation date is required")
    private LocalDateTime evaluationDate;

    @Size(max = 2000)
    private String evaluationNotes;

    @Size(max = 1000)
    private String serviceAdjustments;

    @NotNull(message = "Member ID is required")
    private Long memberId;

    // 🔹 New: Optional for order-linked evaluations
    private Long orderId;

    private Integer rating;

    private String comment;
}
